index.md

```diff
@@ -0,0 +1,31 @@
+# Newdle
+
+**Newdle** is a tool developed by the [Indico Team](https://getindico.io/about/) @ [CERN](https://home.cern/) to **manage the process of organizing meetings**. As other commercial and Open Source solutions, Newdle provides the functionality of creating ad-hoc polls to schedule meetings when everybody (or at least most of the people involved) are able to attend. How is it possible? The tool provides for the creators of the polls (henceforth **newdles**) the possibility to choose different options to schedule the meeting, i.e. to choose different time slots within a day or from several days, in order to find the best options for everybody.
+
+<img src="images/logo_color.svg" alt="hi" width="300" align="center" />
+
+Newdle has been created to streamline the process of choosing **the perfect date and time** for a meeting or event, avoiding the tedious and time consuming tasks that supposes organizing these meetings when is doing sending emails or private messages.
+
+Once the newdle is done, the creator **sends the invitation to vote** to all the participants they want. Those participants will receive an email with the link of the newdle to vote and they can **choose those option(s) that fit** better for them. Once everyone has voted or the creator has closed the newdle it is **easy to choose the final date for the event**.
+
+One of the **most remarkable features** of this tool among others is the **user-friendly and modern interface** and also the additional **freedom and flexibility** that come with being part of an Open Source ecosystem. Additionally, none of them seem to seamlessly integrate with other enterprise systems.
+
+Newdle is part of the [MALT project](https://malt.web.cern.ch/malt/).
+
+![A sneak peek of Newdle](images/sneakpeek.gif)
+
+
+## Main features
+
+- Add participants searching by name and/or email address
+- Choose time slots for different days
+- Copy time slots from previous day
+- Keep list of participants private
+- Check if I am part of any newdles
+- Check my newdles
+
+
+|||
```

Comment:
If we want to keep this thing, we should remove the table. I would just remove it for now.

Reply:
Table removed